src/turnkeyml/llm/tools/ryzenai_npu/ryzenai_npu.py

```diff
@@ -137,6 +144,21 @@ def run(self, state: State, input: str = "", device=None) -> State:
                 f"quantized_llama-2-7b-chat_w{w_bit}_g{group_size}_{algorithm}.pth",
             )
 
+        if (
+            checkpoint == "meta-llama/Meta-Llama-3-8B-Instruct"
+            or checkpoint == "meta-llama/Meta-Llama-3-8B"
+        ):
+            model_name = checkpoint.replace("meta-llama/", "")
+            algorithm = "awqplus"
+            flash_attention_plus = False
+            trust_remote_code = False
+            CausalLMModel = LlamaModelEval
+            LMTokenizer = PreTrainedTokenizerFast
+            quantized_model_path = os.path.join(
+                quantized_models_path,
+                f"quantized_llama-2-7b-chat_w{w_bit}_g{group_size}_{algorithm}.pth",
```

Comment:
should string be `quantized_llama-3-8b`?

Reply:
yup, fixed https://github.com/onnx/turnkeyml/pull/225/commits/c05d6295d931f9811858842b7109dd798369f0c2